src/Compilers/CSharp/Portable/Symbols/AnonymousTypes/SynthesizedSymbols/AnonymousType.DelegateTemplateSymbol.cs

```diff
@@ -18,10 +18,10 @@ internal sealed class AnonymousDelegateTemplateSymbol : AnonymousTypeOrDelegateT
             private readonly ImmutableArray<Symbol> _members;
 
             /// <summary>
-            /// True if any of the delegate parameter types or return type are
-            /// fixed types rather than type parameters.
+            /// True if name of the delegate has to be generated during compilation (&lt;&gt;f__AnonymousDelegate0, 1, ...)
+            /// instead of being fully determined from delegate type (&lt;&gt;A, &lt;&gt;F).
```

Comment:
Consider if the comment can be clarified. The way I see it, both kinds of names are generated during compilation. I think it would be worth to explain the difference between the delegate types. The difference between names doesn't really explain why the types should be handled differently. Or what "properties" those types have by comparison to each other. At least the old property name "HasFixedTypes" was about the delegate type, not about the name.

Reply:
> both kinds of names are generated during compilation

You're right. I clarified the phrasing.

> At least the old property name "HasFixedTypes" was about the delegate type, not about the name.

Yes, but it wasn't used that way. Users of that field were using it to determine whether the delegate has name fully determined just from signature or one derived from source order.

> I think it would be worth to explain the difference between the delegate types.

From the above, I believe keeping this about delegate names is appropriate.